scripts/tests/consistency.sh

```diff
@@ -3,14 +3,14 @@ set -e
 
 case $1 in
     "genesis")
-        cargo run --release --bin kagami -- genesis generate --executor executor.wasm --wasm-dir libs --genesis-public-key ed01204164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4 | diff - defaults/genesis.json || {
-            echo 'Please re-generate the default genesis with `cargo run --release --bin kagami -- genesis --executor executor.wasm --wasm-dir libs --genesis-public-key ed01204164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4 > ./defaults/genesis.json`'
+        cargo run --package iroha_kagami --bin kagami --release -- genesis generate --executor executor.wasm --wasm-dir libs --genesis-public-key ed01204164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4 | diff - defaults/genesis.json || {
```

Comment:
it seems that the unfortunate consequence of defining a root crate `iroha_tests` now hinders our use of `cargo run --bin`. This is quite weird